src/libs/actions/OnyxDerived/configs/reportAttributes.ts

```diff
@@ -73,13 +73,13 @@ export default createOnyxDerivedValueConfig({
                 dataToIterate = prepareReportKeys(updates);
                 recentlyUpdated = updates;
             } else if (!!transactionsUpdates || !!transactionViolationsUpdates) {
-                let transactionReportIds: string[] = [];
+                let transactionReportIDs: string[] = [];
```

Comment:
Addressing this NAB comment from previous PR https://github.com/Expensify/App/pull/61376#discussion_r2082529889 here cc @mountiny